src/EditorFeatures/CSharpTest/ChangeSignature/RemoveParametersTests.cs

```diff
@@ -291,22 +292,13 @@ class C
 
                 var textView = workspace.Documents.Single().GetTextView();
 
-                var handler = new CSharpChangeSignatureCommandHandler(TestWaitIndicator.Default);
-                var delegatedToNext = false;
-                CommandState nextHandler()
-                {
-                    delegatedToNext = true;
-                    return CommandState.Unavailable;
-                }
+                var handler = new CSharpChangeSignatureCommandHandler();
 
-                var state = handler.GetCommandState(new Commands.RemoveParametersCommandArgs(textView, textView.TextBuffer), nextHandler);
-                Assert.True(delegatedToNext);
-                Assert.False(state.IsAvailable);
+                var state = handler.GetCommandState(new RemoveParametersCommandArgs(textView, textView.TextBuffer));
+                Assert.True(state.IsUndetermined);
```

Comment:
which did this change from IsAvailable to IsUndetermined?

Reply:
This test used to verify that in some condition this handler delegates GetComandState() to the next handler and result is CommandState.Unavailable. After the migration, this handler is now a simple non-chained command handler that has no next handler reference. Instead of delegating to the next, it returns CommandState.Undetermined, which indicates that it should be ignored and next handler should be called. So the test was updated to validate this instead.